tests/integration/multicluster/multinetwork/multinetwork_test.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package multinetwork
+
+import (
+	"fmt"
+	"regexp"
+	"strings"
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/environment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/istioctl"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/resource"
+)
+
+var (
+	primary istio.Instance
+	remote  istio.Instance
+	// env     *kube.Environment
+)
+
+// This test requires `--istio.test.env=kube` and at least two clusters on --istio.test.kube.config
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite("multicluster_multi_network_integration_test", m).
+		RequireEnvironment(environment.Kube).
+		Setup(func(ctx resource.Context) (err error) {
+			env := ctx.Environment().(*kube.Environment)
+			if len(env.KubeClusters) >= 2 {
+				return nil
+			}
+			return fmt.Errorf("this test requires 2 KubeConfigs, like  --istio.test.kube.config config-a:config-b")
+		}).
+
+		// Deploy Istio
+		SetupOnEnv(environment.Kube, istio.Setup(&primary, nil)).
+		SetupOnEnv(environment.Kube, istio.Setup(&remote, setupConfigIndex1)).
```

Comment:
note: I am using multiple istio.Setups in https://github.com/istio/istio/pull/21788 for multiple control planes

Reply:
test was removed... resolving.